odm2api/ODM2/services/readService.py

```diff
@@ -155,34 +157,38 @@ def getAnnotations(self, type=None, codes=None, ids=None):
         """
         # TODO What keywords do I use for type.
         a = Annotations
-        if type:
-            if type == 'action':
+        if 'type' in kwargs:
```

Comment:
Should there be a warning if any other kwarg is passed, besides `type`? ie, an unhandled/unrecognized kwarg name.

Not just in this function, but everywhere you use this scheme.

Reply:
I'll see how to best handle unrecognized kwarg